db/migrate/201608152201_create_team_membership_invites_table.rb

```diff
@@ -0,0 +1,25 @@
+class CreateTeamMembershipInvitesTable < ActiveRecord::Migration
+  # rubocop:disable Metrics/MethodLength
+  def up
+    create_table :team_membership_invites do |t|
+      t.integer :team_id, null: false
+      t.integer :user_id, null: false
+      t.integer :inviter_id
+      t.boolean :refused, null: false, default: false
+      t.timestamps null: false
```

Comment:
Do you mean the user? Currently when a user accepts and invite the [MembershipInvite is destroyed and a Membership is created](https://github.com/exercism/exercism.io/pull/3068/files#diff-95453501a52e87dcd00cd144408e029cR10).